GitAssignment/src/com/company/Main.java

```diff
@@ -1,8 +1,39 @@
 package com.company;
 
+import java.util.Scanner;
+
 public class Main {
 
+    public static void getSmallest()
+    {
+                System.out.println("Get Smallest");
```

Comment:
Good work,
I think you should make the function independent from input data. I.e. the main function will handle the input for all functions. Just make your function takes an array as a parameter and do your logic then return the results.